marowitz_robyn_lidar_uncertainty.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# <img style="float: left;" src="earth-lab-logo-rgb.png" width="150" height="150" />
+# 
+# # Earth Analytics Education
+
+# # LiDAR Uncertainty at SOAP and SJER NEON sites
+
+# ## The Soap Site
+# ![Soaproot panorama](img/Soaproot_pano.jpeg)
+# Image Credit: National Ecological Observation Network, available at https://www.neonscience.org/field-sites/soap
+
+# ## The SJER Site
+# ![SJER panorama](img/sjer_site.jpeg)
+# Image Credit: National Ecological Observation Network, available at https://www.neonscience.org/field-sites/sjer
+
+# In[3]:
+
+
+import os
+import pathlib
+
+import earthpy as et 
+import pandas as pd
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import rasterstats as rs
+import xarray as xr
+import rioxarray as rxr
+import seaborn as sns
+
+import clean
```

Comment:
good job on the imports.  Local application/library specific imports should be last. You may want to put them in alphabetical order though